src/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from './reducers';
+// import store from './store';
```

Comment:
is this TODO? or not needed?

Reply:
Eventually I'm gonna have its own store file.